webpack.config.js

```diff
@@ -17,7 +17,7 @@ const webpackMerge = require('webpack-merge');
 //
 // TODO: there ought to be a better way to do this.
 //
-const nodePath = process.env.NODE_PATH || path.join(__dirname, 'node_modules');
+const nodePath = (process.env.NODE_PATH && process.env.NODE_PATH.split(path.delimiter)) || [path.join(__dirname, 'node_modules')];
```

Comment:
@peterbe Prettier does not like this line :) We might have excluded this from Prettier in the past?

Reply:
Yeah, it exceeds 80 characters.